### ISSUE TITLE
Change our Environment Variable support to work same as Next.js

### ISSUE THREAD

**Issue by oiojin831** [blitz-js/blitz#953]
Next.js's built-in support for environment variables is different than Blitz.js's environment variables.

Blitz's only works with .env and next supports other extension such as .env.local

**Comment by flybayer** [comment]
Yes! We need to change our env support to be like Next.js.

I _think_ all we need to do is [change this line](https://github.com/blitz-js/blitz/blob/canary/packages/cli/src/index.ts#L4) to not run for `blitz start` and `blitz build`. Then the .env functionality should "fall through" and be handled by Next.

And then also we need to switch that above line to use [dotenv-flow](https://github.com/kerimdzhanov/dotenv-flow).

To summarize:

- All CLI commands should call `dotenv-flow` except for `blitz start` and `blitz build`
